azure-pipelines-official.yml

```diff
@@ -14,35 +14,56 @@ jobs:
 
   variables:
     BuildPlatform: 'Any CPU'
-    InsertTargetBranchFullName: 'lab/d16.0stg'
-    InsertTargetBranchShortName: 'd16.0stg'
+    VisualStudio.MajorVersion: 16
+    VisualStudio.BranchName: 'lab/d16.0stg'
+    VisualStudio.ChannelName: 'int.d16.0stg'
```

Comment:
Will we need to update these values whenever our internal VS insertion switches back and forth between stg and non-stg?

Reply:
I'm not sure. These are used to build bootstrapper that's used for the training.

@jmarolf

Reply:
Whenever we change were we are inserting we should change these.  At a high level these specify the base build of VS to install roslyn on top of.